Sources/BraintreeThreeDSecure/BTThreeDSecurePostBody.swift

```diff
@@ -0,0 +1,96 @@
+import Foundation
+
+// MARK: - BTThreeDSecurePostBody
+struct BTThreeDSecurePostBody: Encodable {
```

Comment:
Nit - can the docstring for this file include the POST path like the other Encodable types? Will hopefully make searching within the repo easier